test.js

```diff
@@ -48,4 +48,18 @@ describe('csslint-stylish', () => {
 
     assert(CSSLint.hasFormat('stylish'), 'csslint should be stylish')
   })
+
+  it('should not report undefined output lines when no filename provided', () => {
+
+    const res = CSSLint.verify('.class {\n  color: red\n}\n')
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).